lexicons/community/atprotocol/map.json

```diff
@@ -0,0 +1,41 @@
+{
+  "lexicon": 1,
+  "id": "community.atprotocol.geomarker.map",
+  "defs": {
+    "main": {
+      "type": "record",
+      "key": "tid",
+      "record": {
+        "type": "object",
+        "required": ["locations"],
+        "properties": {
+          "label": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 800,
+            "maxGraphemes": 400
+          },
+          "description": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 600,
+            "maxGraphemes": 1200
+          },
+          "mainEntry": {
+            "type": "string",
+            "format": "at-uri",
+            "description": "The main entry associated with this geographic marker."
+          },
+          "locations": {
+            "type": "array",
+            "items": {
+              "type": "ref",
+              "ref": "community.atprotocol.geomarker.marker#main"
```

Comment:
I would personally call these "`markers`" or "`geomarkers`", so it's clear that it refers to our `marker` type. Slight preference toward `marker` since it's the name of the item.